app/views/shared/_eligible_region_content.html.erb

```diff
@@ -1,100 +1,120 @@
-      <li>the age ranges and subjects you’re qualified to teach</li>
-      <li>that you’re qualified to teach at state or government schools.</li>
-    </ul>
-  <% elsif region.status_check_written? %>
-    <p class="govuk-body">The authority or territory that recognises you as a teacher must confirm:</p>
 
-    <ul class="govuk-list govuk-list--bullet">
-      <li>that you’ve completed a course of initial teacher training (ITT)</li>
-      <li>that you’ve successfully completed any period of professional experience comparable to an induction period (if required)</li>
-      <li>the age ranges and subjects you’re qualified to teach.</li>
-    </ul>
-  <% elsif region.sanction_check_written? %>
-    <p class="govuk-body">The education department or authority must also confirm in writing:</p>
-
-    <ul class="govuk-list govuk-list--bullet">
-      <li>that your authorisation to teach has never been suspended, barred, cancelled, revoked or restricted, and that you have no sanctions against you.</li>
-    </ul>
-  <% end %>
-
-  <% if region.status_check_written? || region.sanction_check_written? %>
-    <p class="govuk-body">You’ll need to provide a <span lang="<%= region.country.code %>"><%= region.teaching_authority_certificate %></span>, which you can obtain by contacting:</p>
-
-    <% if region.teaching_authority_address.present? %>
-      <p class="govuk-body app-teaching-authority-address" lang="<%= region.country.code %>"><%= region.teaching_authority_address %></p>
-    <% end %>
-
-    <% if region.teaching_authority_email_address.present? %>
-      <p class="govuk-body"><a class="govuk-link" href="mailto:<%= region.teaching_authority_email_address %>"><%= region.teaching_authority_email_address %></a></p>
-    <% end %>
-
-    <% if region.teaching_authority_website.present? %>
-      <p class="govuk-body">You’ll need to complete the <a class="govuk-link" href="<%= region.teaching_authority_website %>"><span lang="<%= region.country.code %>"><%= region.teaching_authority_certificate %></span> request form</a>.</p>
-    <% end %>
-
-    <p class="govuk-body">This must be dated within 3 months of you applying for QTS.</p>
-  <% end %>
-
-  <% if region.sanction_check_none? %>
-    <p class="govuk-body">You also need to show evidence of your work history.</p>
-
-    <p class="govuk-body">For each school this should include:</p>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>school name</li>
-      <li>school city</li>
-      <li>school country</li>
-      <li>your job role</li>
-      <li>start date</li>
-      <li>end date.</li>
-    </ul>
-  <% end %>
-<% end %>
+<h2 class="govuk-heading-m">What we’ll ask for</h2>
+
+<p class="govuk-body">You’ll need to provide the following evidence when you apply.</p>
+<div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
```

Comment:
There's a helper for accordions in `govuk-components` which might make this code nicer: https://govuk-components.netlify.app/components/accordion/#input-erb-creating-a-simple-accordion

I imagine it would be something like:

```rb
<%= govuk_accordion do |accordion| %>
  <%= accordion.section(heading_text: "1. Proof of identity") do %>
    <p>...</p>
  <% end %>

  <%= accordion.section(heading_text: "2. Proof of qualifications") do %>
    <p>...</p>
  <% end %>
<% end %>
```